spec/rspec/core/suite_hooks_spec.rb

```diff
@@ -23,6 +23,27 @@ module RSpec::Core
           }.to raise_error(ZeroDivisionError)
         end
 
+        it 'runs in the context of an example group' do
+          run_context = nil
+          RSpec.configuration.__send__(registration_method, :suite) { run_context = self }
+          RSpec.configuration.with_suite_hooks { }
+          expect(run_context).to be_a ExampleGroup
+        end
+
+        it 'allows access to rspec-mocks methods within the hook' do
+          run = false
+          RSpec.configuration.__send__(registration_method, :suite) { double('something'); run = true }
```

Comment:
I don't care much.  Mostly I was surprised/confused by it.  However, given I as surprised by it and rspec-mocks features don't work in `:suite` hooks normally without the scope, I think it's worth adding it to prevent future confusion.